test/test.py

```diff
@@ -517,6 +518,17 @@ def test_analysis_protocol(self):
         self.assertEqual(analysis_protocols[0].protocol_id, 'smartseq2_v2.2.0')
         self.assertEqual(analysis_protocols[0].protocol_name, None)
 
+    def test_imaging_protocol(self):
+        uuid = '94f2ba52-30c8-4de0-a78e-f95a3f8deb9c'
+        version = '2019-04-03T103426.471000Z'
+        manifest, metadata_files = self._load_bundle(uuid, version, replica='aws', deployment='staging')
+        bundle = Bundle(uuid, version, manifest, metadata_files)
+        imaging_protocols = [p for p in bundle.protocols.values() if isinstance(p, ImagingProtocol)]
+        self.assertEqual(len(imaging_protocols), 1)
+        self.assertEqual(len(imaging_protocols[0].target), 240)
```

Comment:
first() or one() from more_itertools instead of `[0]`?

Reply:
more_itertools is not currently imported into the metadata-api project. If it is ok to add the import of the module I will do so.

Reply:
I wouldn't want it as a main dependency, but as a test dependency it should be ok.